io/plugins/eu.esdihumboldt.hale.io.gml.test/src/eu/esdihumboldt/hale/io/gml/writer/internal/StreamGmlWriter2Test.groovy

```diff
@@ -381,4 +381,50 @@ class StreamGmlWriter2Test {
 
 		result
 	}
+
+	@Test
+	void testNumbers() throws Exception {
+		testNumber("integerNumber", 12345)
+		testNumber("doubleNumber", 12345.123456789)
+		testNumber("floatNumber", 12345.6789F)
+		testNumber("longNumber", 1234567890123456789L)
+		testNumber("shortNumber", 12345 as short)
+		testNumber("decimalNumber", new BigDecimal("1234567890.123456789"))
```

Comment:
Would any of these tests before your changes resulted in a scientific notation being used?

In any case I think it would be good to have a test that explicitly uses a scientific notation as input value.

Reply:
Done.